S26-documentation/07a-tables.t

```diff
@@ -88,6 +88,7 @@ is $r.contents[1].elems, 3;
 is $r.contents[2].elems, 3;
 
 # test fix for RT #132341
+# also tests fix for RT #129862
```

Comment:
what do you mean by that? The test was passing before, so what changed? Was this line intended to be here? If yes, then ok.

Reply:
RT #129862 was fixed by a downstream change to Pod::To::Text. The new NQP Pod changes fix that original problem at the source in the Pod code.